pom.xml

```diff
@@ -67,7 +67,7 @@
 
   <properties>
     <netty.version>5.0.0.Alpha5</netty.version>
-    <netty.build.version>30</netty.build.version>
+    <netty.build.version>29</netty.build.version>
```

Comment:
Downgrading? Do you have plans for a round of dependency upgrades later?

Reply:
thanks for reviewing;

so, I have aligned with other projects as close as possible; and all other netty-contrib projects are using netty.build.version=29, so, I then downgraded from 30 to 29 (initially, when I created the project, I used netty.build.version=30, but honestly, I don't remember why :thinking:)

I see that latest version of io.netty:netty-build-common is 31, so maybe better is to use the latest version right ?

Reply:
Yeah. I don't know what's changed, but I assume newer is better. We can make the changes one at a time, just in case something breaks.